### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by pastorp3**
##What this PR for :

- Build it with CSS, HTML and Bootstrap

- linters working 

-clone according to the original page

### CODE REVIEW

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 
Hey @pastorp3  and @sanadwj 

You did a very good job on this project. The design looks quite good, the code clean, and you displayed a great knowledge of the bootstrap classes.

However, there are still a few aspects that we need to correct in order to advance to the next project:

- [ ] Do not commit unnecessary files (like the node_modules). Please remove them from GitHub for the next review.  I saw that they are on git ignore. Delete them, do a commit, and run npm install again (this should fix the problem).

#### Navbar 

- [x] The subscribe button is overflowing to the next line on the medium screen.

| Yours | Original |
|:---:|:---:|
| image|image  |

- [x] Newsweek logo is too small on mobile screen size.

| Yours | Original |
|:---:|:---:|
|image |image  |

#### The debate
- [x] Try to make the styling more like the original. This applies to the whole page, especially the fonts weights and size.

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] On the mobile size, the size of the image is not proportional.

| Yours | Original |
|:---:|:---:|
|image |image  |


#### More Stories

- [x] The text should wrap around the images.  (All screen sizes)

| Yours | Original |
|:---:|:---:|
|image |image  |

- [ ] In the tablet screen, the images are distorted. This happens in different places as well. Address that for the next review.

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] The padding on the medium screen is too much.

| Yours | Original |
|:---:|:---:|
|image |image |

- [x] Also in the mobile screen

| Yours | Original |
|:---:|:---:|
|image | image |

#### In the magazine

- [x] In the mobile screen, the images need to be centered.

| Yours | Original |
|:---:|:---:|
|image |image |

#### Bottom news section

- [x] The layout needs to be closer to the original. Pay extra attention to the fonts and images sizes.

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] Also on tablet and mobile


| Yours | Original |
|:---:|:---:|
|image | image |

#### Subscribe && Newsletter

- [x] Pay attention to the layout in the mobile screen size. (Don't mind the navbar in the screenshot)

| Yours | Original |
|:---:|:---:|
| image|image |

#### Footer

- [x] Re-check the layout in all screen sizes again.

| Yours | Original |
|:---:|:---:|
|image | image |

#### Even if it seems a lot, it is not. This project is very long and tiresome, and you really did a great job :1st_place_medal: 

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by pastorp3** [comment]
- add all changes

- linters working

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far, team. 👍 

- Good work wrapping the text around the images as the previous TSE requested.

## However, there are some changes required 👀 🧰 🔨 🔧 

- Please update your Mobile view to fix this overflow of the hamburger and subscribe button:
image

- The change requested on "The Debate" section from the previous TSE has not been implemented. Please fix this:
image

- Your page overflows to the sides on Tablet view. Please fix this.
image

- Please center the images on mobile view like the previous TSE requested.
image

- On width 1024px you have a porblem with overflowing text on the right side articles. Please fix this as well:
image

- Also, please fix your footer on mobile view:

| Your version | Original |
| --- | --- |
| image | image |

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by pastorp3** [comment]
- add changes

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on some changes before we can merge.

- The elements in the navbar should be vertically aligned.

  image

- Please fix this 👇 in the more stories section

  image

- The magazine and the text below should have the same width.

  | Source Page| Clone Page |
  |:---:|:---:|
  |image|image|

- The images here must have the same height and width.

  image

  _Possible solution: set image `height` and `width` and give them a `object-fit: cover`_

- The texts are still overflowing, as pointed out by the previous TSE 👇 

  image

- The subscribe and newsletter section is still misaligned 👇 

  | Source Page| Clone Page |
  |:---:|:---:|
  |image|image


- The footer is still misaligned 👇 

  image

- The navbar should be fixed in tablets and mobile devices.

  | Source Page| Clone Page |
  |:---:|:---:|
  |image|image|

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Comment by pastorp3** [comment]
- add review changes

- we try to fix the overflow in text but we cant see it in the live server

- already add width and height and also fit cover to img and still the same

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+    margin-left: -95px !important;
+  }
+
+  .card-db-txt-special {
+    margin-left: 55px !important;
+    margin-top: 10px !important;
+  }
+
+  .vs {
+    left: 350px !important;
```
Comment (andresporras3423) [inline styles.css]: Don't overuse important

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+  right: 94px;
+}
+
+.nav-md {
+  display: none;
+}
+
+.nav-sm {
+  display: none;
+}
```
Comment (andresporras3423) [inline styles.css]: Group selectors with the same style definition

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+    left: 350px !important;
+  }
+
+  .copy-m {
+    margin-top: 20px;
+  }
+
+  .icon-m {
+    margin-top: 20px;
+  }
```
Comment (andresporras3423) [inline styles.css:312]: Group selectors with the same style definition

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+    padding: 0 0 0 0 !important;
+  }
+
+  .more-stories {
+    padding: 0 0 0 0 !important;
+  }
+
+  .padding-op {
+    padding: 0 0 0 0 !important;
+  }
```
Comment (andresporras3423) [inline styles.css]: Group selectors with the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @sanadwj @pastorp3** 
Good job with your new submission for this project :muscle:  :muscle:
Some improvements are required before you can go ahead:
- [x] Don't overuse important
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can try creating new classes.
- [x] remove the horizontal scrollbar
- [x]  Check out the "opinion" section and compare with the original, make sure to match the original structure:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] in the bottom of your page, math the appearance of the original "subscribe" section:

| Yours | Original |
|:---:|:---:|
|image | image |

### OPTIONAL

- [x] try to use footer and main tags in your page

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+  right: 94px;
+}
+
+.nav-md {
+  display: none;
+}
+
+.nav-sm {
+  display: none;
+}
```
Comment (kubilaycaglayan) [inline styles.css]: Group selectors with the same style definition(same as the previous TSE requested)
You can easily do this by:
``` 
.dis-none {
  display: none;
}
```

And you can use **dis-none** class name in any element you want.

Moreover, you have a built-in Bootstrap class name for this, d-none

Check this life-changing [link](https://getbootstrap.com/docs/4.5/utilities/display/#hiding-elements)

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team :wave: 
Status: Changes Required :recycle: 

Thank you very much for all your efforts :muscle: , looks like your project becoming better and better. :star: 
:battery:  Great opinion section implementation!

- [x]  Please try, one more time, to delete overused "!important" rules. In styles.css file, we have 50 of them.

- [x]  Grouping selectors is an important skill in CSS, as the previous TSE mentioned, try to group your selectors which has the same rules. I wrote an explanation, please check below.

- [ ]  In the subscribe section, those two magazine pictures should be on the different lines, check the previous TSE's comment to see the original, please. 

:camera_flash:  **Screenshot**

subscribe section

That's it...
With all my respect...

**Comment by pastorp3** [comment]
- add review changes

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

The page looks really great, and you did a great job on everything that was requested above. The page looks much better than when I first reviewed :1st_place_medal: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**